feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go

```diff
@@ -349,15 +353,15 @@ func TestWrrTraffic(t *testing.T) {
 		"intf1-af3": {
 			frameSize:             1000,
 			trafficRate:           40,
-			expectedThroughputPct: 100.0,
+			expectedThroughputPct: 83.48,
```

Comment:
Can you explain your calculation?

We may need to calculate this dynamically based on the weights.